org.eyecreate.qiflora.json

```diff
@@ -0,0 +1,120 @@
+{
+    "id": "org.eyecreate.qiflora",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.12",
+    "sdk": "org.kde.Sdk",
+    "command": "qiflora",
+    "tags": ["kde"],
+    "finish-args": [
+        "--share=ipc",
+        "--allow=bluetooth",
+        "--system-talk-name=org.bluez",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "separate-locales": false,
+
+    "modules": [
+        {
+            "name": "udev",
```

Comment:
You could remove udev if you can bump the runtime to 5.13

Reply:
I tried using the newer runtime, but it said it couldn't find udev. I reverted it back to what works.